libc/test/src/math/RoundToIntegerTest.h

```diff
@@ -26,11 +26,11 @@ using LIBC_NAMESPACE::Sign;
 static constexpr int ROUNDING_MODES[4] = {FE_UPWARD, FE_DOWNWARD, FE_TOWARDZERO,
                                           FE_TONEAREST};
 
-template <typename F, typename I, bool TestModes = false>
+template <typename F, typename OutType, bool TestModes = false>
```

Comment:
@lntue , could we not also have the template def here as : 

```
template <typename FloatType, typename IntType,
```

Reply:
sure